package.json

```diff
@@ -3,22 +3,32 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@coreui/coreui-pro": "^2.1.14",
+    "@coreui/icons": "^0.3.0",
+    "@coreui/react": "^2.5.2",
     "@types/jest": "24.0.15",
     "@types/node": "12.6.8",
     "@types/react": "16.8.23",
     "@types/react-bootstrap": "^0.32.19",
     "@types/react-dom": "16.8.4",
     "@types/react-redux": "^7.1.1",
     "@types/react-router-dom": "^4.3.4",
+    "@types/reactstrap": "^8.0.4",
```

Comment:
we prefer react-bootstrap over reactstrap
https://www.npmtrends.com/react-bootstrap-vs-reactstrap